eastlake/steps/newish_metacal/metacal/metacal_fitter.py

```diff
@@ -159,7 +138,73 @@ def _do_all_metacal(self, mbobs_list, data=None):
         output = eu.numpy_util.combine_arrlist(datalist)
         return output
 
-    def _do_one_metacal(self, mbobs):
+    def _do_one_metacal_ngmixv2(self, mbobs):
+        gm = GaussMom(1.2).go(mbobs[0][0])
+        if gm['flags'] == 0:
+            flux_guess = gm['flux']
+            Tguess = gm['T']
+            guesser = TFluxAndPriorGuesser(
+                rng=self.rng, T=Tguess, flux=flux_guess, prior=self.metacal_prior,
+            )
+        else:
+            guesser = PriorGuesser(self.metacal_prior)
+        psf_guesser = SimplePSFGuesser(rng=self.rng)
+
+        fitter = Fitter(
+            model=self['metacal']['model'],
+            fit_pars=self['metacal']['max_pars']['pars']['lm_pars'],
+            prior=self.metacal_prior
+        )
+        psf_fitter = Fitter(
+            model=self['metacal']['psf']['model'],
+            fit_pars=self['metacal']['psf']['lm_pars'],
+        )
+
+        psf_runner = PSFRunner(
+            fitter=psf_fitter,
+            guesser=psf_guesser,
+            ntry=self['metacal']['psf']['ntry'],
+        )
+        runner = Runner(
+            fitter=fitter,
+            guesser=guesser,
+            ntry=self['metacal']['max_pars']['ntry'],
+        )
+
+        boot = MetacalBootstrapper(
```

Comment:
Is to run standard metacal as an alternative to metadetect?

Reply:
Note that ` BootPSFFailure` exception can be raised.

Reply:
Yes. This code is used to check the sims when we have grid around.

Reply:
Errors are caught by the caller of this method.